src/css/_zendesk-category-section.scss

```diff
@@ -7,8 +7,10 @@
 		padding: 0;
 
 		.article-list-item {
+			border-bottom: 1px solid $color-border;
 			list-style: none;
-			padding-bottom: $padding-default;
+			margin-bottom: 1rem;
+			padding-bottom: 1rem;
```

Comment:
Can we make a variable for this like `$margin-medium` and `$padding-medium` like we do for `$margin-small` and `$padding-small`?

We would need to make that change throughout.

Reply:
Sure thing.